book-1-kennels/chapters/PK_INTRO_SETUP.md

```diff
@@ -156,18 +152,20 @@ if __name__ == "__main__":
 ```
 
 ## Adjust VS Code Settings
-First you'll need to select the correct Python Interpreter. Open the command palette with `cmd+shift+p` and select "Python: Select Interpretor". Find the option that has `<your folder name>-<random string>`
+
```

Comment:
Headings require a space before and after them in markdown

Reply:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]